muhireheir/mybrand
# Ft deleteblog

> #### What does this PR do?
> * add functionnality  by   which bblog  owner can delete  a blog
> #### Description of Task to be completed
> * delete a  singleblog
> #### How should this be manually tested?
>  * clone the repository
> * change to folder-structure branch
> * go into ui folder
> * open index.html with your browser
> * click on sign in
> * enter email and password for admin(muhire416@gmail.com,Umusaza36)
> * click  on  blogs
> * click  on  bloglist
> * you  will see alist of blogs click on trash icon of anyblog
> * click yes on the confirmationn popup
> * done 
> #### Any background context you want to provide?
> *  to help admin delete a blog if needed without manualy deleting it in   firebase.
> #### What are the relevant Trello story?
>  *Delete an existing article
> #### Screenshots (if appropriate)
>
